lib/manage-prototype-handlers.js

```diff
@@ -42,7 +42,7 @@ async function isValidVersion (packageName, version) {
   const { versions = [], localVersion } = await lookupPackageInfo(packageName, version)
   const isVersionValid = [...versions, localVersion].includes(version)
   if (!isVersionValid) {
-    console.log('version', version, ' is not valid, valid options are:\n\n', versions)
+    console.log('version', version, ' is not valid, valid options are:\n\n', localVersion || versions)
```

Comment:
This log doesn't describe the condition if `localVersion` is set.  I'd suggest using the same array as you check against (`const validVersions = [...versions, localVersion]`)